src/server/controllers/paypal.controller.ts

```diff
@@ -0,0 +1,18 @@
+import { createBuzzOrder, processBuzzOrder } from './../services/paypal.service';
+import { Context } from '~/server/createContext';
+
+import { PaypalPurchaseBuzzSchema, PaypalOrderSchema } from '../schema/paypal.schema';
+
+export const createBuzzOrderHandler = async ({
+  input,
+  ctx,
+}: {
+  input: PaypalPurchaseBuzzSchema;
+  ctx: DeepNonNullable<Context>;
+}) => {
+  return await createBuzzOrder({ userId: ctx.user.id, ...input });
```

Comment:
Maybe wrap in try/catch to handle the errors correctly? Seems like in the service itself we are not capturing some potential places where promises might fail